generatorreceiver/internal/flags/flag.go

```diff
@@ -139,3 +147,23 @@ func (f *Flag) parent() *Flag {
 	}
 	return Manager.GetFlag(f.cfg.Incident.ParentFlag)
 }
+
+func (ic IncidentConfig) validate() error {
+	if Manager.GetFlag(ic.ParentFlag) == nil {
+		return fmt.Errorf("parent flag %s does not exist", ic.ParentFlag)
+	}
+	if len(ic.Start) == 0 {
+		return fmt.Errorf("start cannot be empty")
+	}
+	if ic.Duration == 0 && len(ic.Start) > 1 {
+		return fmt.Errorf("if duration is not specified, only one start time is permitted (will last until end of incident)")
```

Comment:
Technically we don't need this validation step. If multiple start times were specified without a duration set, we could  just have the child flag be enabled from the first start time until the end of the incident. However I included this validation step anyway because it might be unclear to users editing YAML files as to what the expected behavior is if multiple start times are specified but no duration.